backend/routes/index.js

```diff
@@ -0,0 +1,174 @@
+var express = require('express');
+var router = express.Router();
+
+//Using request for super simple api requests
+var request = require('request');
+
+//Read our keys.json file
+var fs = require("fs");
+
+//Parse our keys.json
+keys = JSON.parse(fs.readFileSync('keys.json', 'utf8'));
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  res.render('index', { title: 'I am a Hamburger' });
```

Comment:
I think he's just cracking up at the throwaway response to someone accessing /

Personally, I like returning HTTP 418 in this scenario :)